__tests__/app.test.js

```diff
@@ -93,9 +94,27 @@ describe("/api/articles/:article_id", () => {
   });
 });
 
-  
-  
+describe('/api/articles', () => {
+  test('GET:200 - responds with all the articles', () => {
+    return request(app)
+      .get('/api/articles')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles).toHaveLength(13);
 
-  
+        body.articles.forEach(article => {
+          expect(Object.keys(article).length).toBeGreaterThan(0); 
```

Comment:
This test is unecessary, by testing the keys below you will ensure the object has more than 0 keys